config/initializers/1_imgproxy.rb

```diff
@@ -1,3 +1,5 @@
+# This file has to run before Carrierwave
```

Comment:
I need this file to run first so CarrierWave can use `Images::Optimizer.imgproxy_enabled?` properly.

Reply:
🔥